addons/web/static/src/actions/action_service.js

```diff
@@ -226,7 +226,13 @@ function makeActionManager(env) {
     }
     if (action.controllers[view.type]) {
       // this controller has already been used, re-import its exported state
-      props.state = action.controllers[view.type].exportedState;
+      if ("exportedState" in action.controllers[view.type]) {
+        debugger
```

Comment:
bug here --> willUnmount to late --> use event "ACTION_MANAGER:EXPORT_CONTROLLER_STATE"